src/Controls/src/Core/Layout.cs

```diff
@@ -108,6 +108,8 @@ public bool CascadeInputTransparent
 			set => SetValue(CascadeInputTransparentProperty, value);
 		}
 
+		IReadOnlyList<Element> IElementController.LogicalChildren => InternalChildren;
```

Comment:
This keeps anyone accessing LogicalChildren through `IElementController` consistent. This should really only be from compatibility renderers